multimedia/v4l2camera/Makefile

```
@@ -0,0 +1,87 @@
+# This software is in the public domain, furnished "as is", without technical
+# support, and with no warranty, express or implied, as to its usefulness for
+# any purpose.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=v4l2camera
+PKG_VERSION:=0.1.8
+PKG_RELEASE:=2
```

```suggestion
PKG_RELEASE:=1
```

this is a new package, so set it to `1`